recipe-server/normandy/base/api/routers.py

```diff
@@ -0,0 +1,34 @@
+from django.conf.urls import url
+
+from rest_framework.routers import DefaultRouter
+
+from normandy.base.api.views import APIRootView
+
+
+class RouterWithDetachedViews(DefaultRouter):
+    """Router that allows for detached routes."""
+
+    def __init__(self, *args, view=APIRootView, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.view = view
+        self.registered_view_urls = []
+
+    def get_urls(self):
+        """
+        Generate the list of URL patterns, including a default root view for the API.
+        """
+        # Get the superclass of DefaultRouter, skipping over DefaultRouter
```

Comment:
`get_urls` is where the format suffix magic of `DefaultRouter` happens, so if you're skipping over it, you might as well just subclass `SimpleRouter` directly.